src/directives/for.js

```diff
@@ -14,7 +14,13 @@ module.exports = {
     // support "item in items" syntax
     var inMatch = this.expression.match(/(.*) in (.*)/)
     if (inMatch) {
-      this.alias = inMatch[1]
+      var itMatch = inMatch[1].match(/\((.*),(.*)\)/)
+      if (itMatch) {
+        this.iterator = itMatch[1]
```

Comment:
Small thing: we should call `trim()` on the iterator and alias here so that cases with extra spaces like `( key, val )` would also work.